app/utils/affiliate.js

```diff
@@ -0,0 +1,50 @@
+import config from '../config/environment';
+
+function getServiceHost() {
+  if (config.environment === 'production') {
+    return 'https://services.fandom.com/affiliate/redirect/'; // add r=
+  }
+
+  return 'https://services.fandom-dev.us/affiliate/redirect/';
+}
+
+export function trackingPairsToObject(unit) {
+  const tracking = {
+    algo: '',
+    method: '',
+    version: '',
+    recommendation_level: '',
+  };
+
+  if (unit.tracking && unit.tracking.forEach && typeof unit.tracking.forEach === 'function') {
+    unit.tracking.forEach((kv) => {
+      tracking[`${kv.key}`] = kv.val;
+    });
+  }
+
+  return tracking;
+}
+
+export function linkToProxyLink(link, unit, wikiId, articleId = -1) {
+  // deploy to one community (marvel)
```

Comment:
I'm curious why marvel needs a link to proxy link?

Reply:
We are just doing a limited roll out. We will be doing this for every community.